linkedin.py

```diff
@@ -72,8 +72,11 @@ def linkJobApply(self):
 
         for url in urlData:        
             self.driver.get(url)
-
-            totalJobs = self.driver.find_element(By.XPATH,'//small').text 
+            try:
+                totalJobs = self.driver.find_element(By.XPATH,'//small').text 
+            except:
+                print("No Matching Jobs Found")
+                continue
```

Comment:
Seems good, I tested it and run into some errors, but I will approve it for now. Thank you for your contribution @cheseremtitus24 !